tests/playbooks/tests_wireless_exists.yml

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  tasks:
+    - name: remove the NetworkManager-wifi package
+      package:
+        name: "NetworkManager-wifi"
+        state: absent
+
+    - name: "get the rpm package facts"
+      package_facts:
+        manager: "auto"
+
+    - name: "NetworkManager-wifi checking before wireless configuration"
+      debug:
+        msg: "NetworkManager-wifi is removed before wirelss configuration"
+      when: "'NetworkManager-wifi' not in ansible_facts.packages"
+
+    - name: "wireless configuration"
+      include_role:
+        name: linux-system-roles.network
+      vars:
+        network_connections:
+          - name: wlan0
+            type: wireless
+            wireless:
+              ssid: "My WPA2-PSK Network"
+              key_mgmt: "wpa-psk"
+            # recommend vault encrypting the wireless password
+            # see https://docs.ansible.com/ansible/latest/user_guide/vault.html
+              password: "p@55w0rD"
+
+    - name: "get the rpm package facts"
+      package_facts:
+        manager: "auto"
+
+    - name: "NetworkManager-wifi checking after wireless configuration"
+      debug:
+        msg: "NetworkManager-wifi is installed after wireless configuration"
+      when: "'NetworkManager-wifi' in ansible_facts.packages"
```

Comment:
IMO, this should be an assert not a debug message.

Reply:
thanks for hint